README.md

```diff
@@ -29,12 +31,36 @@ Your project will be versioned according to the latest tag found in the commit h
 
 ## Usage
 
-When you want to release a version of your software, whether it's a pre-release, RTM, patch, or anything else, simply create a tag with a name which is a valid [SemVer 2.0](https://semver.org/spec/v2.0.0.html) version and build your projects. MinVer will apply the version to the assemblies and packages. (If you like to prefix your tag names, see the [FAQ](#can-i-prefix-my-tag-names).) Note that MinVer sets `AssemblyVersion` to `{MAJOR}.0.0.0`, but [this behaviour can be overridden](#can-i-use-the-version-calculated-by-minver-for-other-purposes).
+When you want to release a version of your software, whether it's a pre-release, RTM, patch, or anything else, simply create a tag with a name which is a valid [SemVer 2.0](https://semver.org/spec/v2.0.0.html) version and build your projects. MinVer will apply the version to the assemblies and packages. (If you like to prefix your tag names, see the [FAQ](#can-i-prefix-my-tag-names).)
+
+## How it works
+
+When the current commit is tagged with a version, the tag is used as-is.
 
 When the current commit is not tagged, MinVer searches the commit history for the latest tag. If the latest tag found is a [pre-release](https://semver.org/spec/v2.0.0.html#spec-item-9), MinVer will use it as-is. If the latest tag found is RTM (not pre-release), MinVer will increase the patch number and add default pre-release identifiers, e.g. `1.0.0` becomes `1.0.1-alpha.0`. If no tag is found, the default version `0.0.0-alpha.0` is used.
 
 You will notice that MinVer adds another number to the pre-release identifiers when the current commit is not tagged. This is the number of commits since the latest tag, or if no tag was found, since the root commit. This is known as "height". For example, if the latest tag found is `1.0.0-beta.1`, at a height of 42 commits, the calculated version is `1.0.0-beta.1.42`.
 
+## Version numbers
+
+MinVer sets the following custom properties:
+
+- `MinVerVersion`
+- `MinVerMajor`
+- `MinVerMinor`
+- `MinVerPatch`
+
+Those properties are used to set the following .NET SDK properties, satisfying the official [open-source library guidance for version numbers](https://docs.microsoft.com/en-ca/dotnet/standard/library-guidance/versioning#version-numbers):
+
+Property | Value
+-- | --
+`AssemblyVersion` | `{MinVerMajor}.0.0.0`
+`FileVersion` | `{MinVerMajor}.{MinVerMinor}.{MinVerPatch}.0`
+`PackageVersion` | `{MinVerVersion}`
+`Version` | `{MinVerVersion}`

```

Comment:
@bording do you know if setting `Version` is actually required? The [guidance](https://docs.microsoft.com/en-ca/dotnet/standard/library-guidance/versioning#version-numbers) doesn't mention it.

Reply:
`Version` gets used in a number of places in the SDK targets files to derive various other version values, so it does get used, but it doesn't directly map to one of the final outputs.

For example, if you stop setting `Version`, you would have to manually set `InformationalVersion`

Reply:
👍 thanks. What about `PackageVersion`?

Reply:
If we were setting `Version` earlier and not in a target, `PackageVersion` would automatically be derived from `Version` as well.

Reply:
Ah yes, I remember that now from the initial experimentation. 👍